setup.py

```diff
@@ -102,6 +106,11 @@ def main():
         ],
     }
     install_requires = [
+        "0x-contract-addresses",
```

Comment:
Why do we need all of these 0x libraries?

Reply:
can't import from hummingbot.connector.utils without them

Reply:
I guess we could split it off into a specific extras_require ? if it's only really useful for hummingsim users

Reply:
I think that's fine, I just couldn't find where they are used.